### ISSUE TITLE
Respond to prompt omaewamoushindeiru

### ISSUE THREAD

**Issue by wendko** [angch/multibot#7]
Responds to messages containing string
- `お前はもう死んでいる`
- `Omae wa mou shindeiru` (case insensitive)
- `omaewamoushindeiru` (case insensitive)
with
- `何?!`
- `なに?!`
- `Nani?!`
followed by ASCII art
```
     _.-^^---....,,--       
 _--                  --_  
<                        >)
|                         | 
 \._                   _./  
    ```--. . , ; .--'''       
          | |   |             
       .-=||  | |=-.   
       `-=#$%&%$#=-'   
          | ;  :|     
 _____.,-#%&$@%#&#~,._____
```

**Comment by pickfire** [comment]
Fourth, should be `何？！` or the meme

**Comment by angch** [comment]
Fixed: #9
